src/server/scripts/Commands/cs_player.cpp

```diff
@@ -53,15 +58,21 @@ class player_commandscript : public CommandScript, public PlayerCommand
             return false;
 
         char* playerName = strtok((char*)args, " ");
-        char* spellid = strtok(nullptr, " ");
+        char* spellId = strtok(nullptr, " ");
         char const* all = strtok(nullptr, " ");
         Player* targetPlayer = FindPlayer(handler, playerName);
-        if (!spellid || !targetPlayer)
+
+        if (!spellId || !targetPlayer)
+        {
             return false;
+        }
+
+        uint32 spell = handler->extractSpellIdFromLink(spellId);
 
-        uint32 spell = handler->extractSpellIdFromLink(spellid);
         if (!spell)
+        {
             return false;
+        }
 
         return UnLearn(handler, targetPlayer, spell, all);
     }
```

Comment:
`learnCommand` and `unlearnCommand` functions are almost the same, we could think to merge them into one with a `boolean` parameter `learn`

Reply:
for now I did not want to alter the code too much